src/components/PersonaCard/PersonaCard.scss

```diff
@@ -181,29 +182,17 @@
   }
 }
 
-
-//== Modifier: Persona card with square personas
-//
-.ms-PersonaCard.ms-PersonaCard--square {
-  .ms-PersonaCard-persona {
-    .ms-Persona-imageArea,
-    .ms-Persona-image {
-      width: 100px;
-      height: 100px;
-      margin: 0;
-    }
-
-    .ms-Persona-presence {
-      left: 0;
-    }
-  }
-}
-
-
 @media (min-width: $ms-screen-md-min) {
   // Undock the persona card and give it a shadow.
   .ms-PersonaCard {
+    @include drop-shadow;
     max-width: 360px;
     position: relative;
   }
+
+  .ms-ContextualHost {
```

Comment:
This is a great solution for now, but we should discuss whether ContextualHost should be applying to shadow or just positioning the component.